src/transformers/blenderbot-90M-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "embedding_size": 512,
+  "n_layers": 8,
+  "ffn_size": 2048,
+  "dropout": 0.1,
+  "n_heads": 16,
+  "n_positions": 512,
+  "activation": "gelu",
+  "attention_dropout": 0.0,
+  "relu_dropout": 0.0,
+  "learn_positional_embeddings": true,
+  "vocab_size": 54944,
+  "pad_idx": 0,
```

Comment:
We usually use `pad_token_id` (Not 100% if this is the same here, but I think so). `pad_token_id` is also written in the `configuration_utils` file so you only would have to pass it to `__init__()`

Reply:
Yes it's the same. will update the name.